doc/deploy/deployment.md

```diff
@@ -95,10 +95,15 @@ or
 ```
 ### 2. Navigate to the Helm Deployment Directory.
 ```
-cd multi-cluster-app-wrapper/deployment
+cd multi-cluster-app-dispatcher/deployment/mcad-controller
+```
+### 3.a. Optional : Installation using Helm3
```

Comment:
The optional part is using the `quota-management` branch not necessarily using Helm3?  I would recommend this not be optional otherwise we will need to put back the older instructions of Helm2.  Your choice either way.

Reply:
Done, created new section branch checkout.